dev/README.md

```diff
@@ -29,3 +34,15 @@ export MM_ADMIN_PASSWORD=(your Mattermost admin password)
 ```
 
 Then run `make deploy` to compile the Apps plugin from source and automatically redeploy the plugin to your server.
+
+## Mattermost development environment
+
+Alternativly, you can setup a fully-fledged Mattermost development environment by following the steps [here](https://developers.mattermost.com/contribute/server/developer-setup/).
```

Comment:
```suggestion
Alternatively, you can setup a fully-fledged Mattermost development environment by following the steps [here](https://developers.mattermost.com/contribute/server/developer-setup/).
```

Reply:
1/5 This suggestion should be applied

Reply:
It's already applied: https://github.com/mattermost/mattermost-plugin-apps/pull/213/files#diff-e14025c0fa40d4857e4b40fc96ea5ee995afe300014626b68cb55a479fa5b8fcR40